### PR TITLE
Feature/151297883/static page management

### PR DESCRIPTION
**Pull request by ibrahim013**
**What does this PR do?**

This PR implements the feature that allows the admin to add and manage static pages.

**Description of Task to be completed?**
Install required modules
add static page management panel to the admin dashboard.
integrate an end to end test to test functionalities.

**How should this be manually tested?**
Navigate to the admin dashboard and click on the "Static Page Management" panel to open the page to add the static page to be displayed to users.

**What are the relevant pivotal tracker stories?**
#151297883/Static Page Management

Screenshots

screen shot 2017-10-11 at 12 38 51 pm
screen shot 2017-10-11 at 12 38 47 pm
screen shot 2017-10-11 at 12 38 43 pm

### CODE REVIEW

File: client/modules/accounts/templates/dropdown/dropdown.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
 import { Reaction, Logger } from "/client/api";
-import { Tags } from "/lib/collections";
 import { Session } from "meteor/session";
 import { Meteor } from "meteor/meteor";
 import { Template } from "meteor/templating";
+import { Tags, StaticPages } from "./../../../../../lib/collections/collections";
```
Comment (emmyakin) [inline client/modules/accounts/templates/dropdown/dropdown.js]: is this import correct? can't you import /lib/collections/collections?

File: client/modules/router/main.js [hunk 2]
```diff
@@ -4,9 +4,9 @@ import { Meteor } from "meteor/meteor";
 import { Tracker } from "meteor/tracker";
 import { FlowRouter as Router } from "meteor/kadira:flow-router-ssr";
 import { BlazeLayout } from "meteor/kadira:blaze-layout";
-import { Reaction, Logger } from "/client/api";
-import { Packages, Shops } from "/lib/collections";
-import { MetaData } from "/lib/api/router/metadata";
+import { Reaction, Logger } from "../../api";
+import { Packages, Shops } from "../../../lib/collections";
+import { MetaData } from "../../../lib/api/router/metadata";
```
Comment (emmyakin) [inline client/modules/router/main.js:9]: probably should not have relative imports like this
use /lib/....

File: imports/plugins/core/layout/client/templates/layout/staticPages/staticPage.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { StaticPages } from "../../../../../../../../lib/collections/collections";
```
Comment (emmyakin) [inline imports/plugins/core/layout/client/templates/layout/staticPages/staticPage.js]: same here

File: .env [hunk 4]
```diff
@@ -0,0 +1,3 @@
+REACTION_EMAIL=amtmgs7t@localhost
```
Comment (emmyakin) [inline .env]: this again?

**Review by emmyakin — CHANGES_REQUESTED** [review]
Please rebase your branch

File: client/modules/router/main.js (hunk 2 above)
Comment (ibrahim013) [reply client/modules/router/main.js:9]: when i try to do that the app keeps breaking

**Review by emmyakin — APPROVED** [review]

**Pull request merged**